openedx/features/edly/utils.py

```diff
@@ -0,0 +1,18 @@
+from openedx.features.edly.models import EdlyUserProfile
+
+
+def user_has_edly_organization_access(request):
+    """
+    Return True if the edly-sub-organization related to request URL is listed in get_linked_edly_sub_organizations.
+    """
+    full_url = request.build_absolute_uri('?')
+    edly_profile = EdlyUserProfile.objects.filter(user=request.user).first()
```

Comment:
use reverse relation with `request.user`and related name `edly_profile`

Reply:
Done